### DISCUSSION TITLE
Wrong reference to GoogleService-Info.plist?

### DISCUSSION THREAD

**Discussion by MatiSera** [invertase/react-native-firebase#4703] [Q&A] [answered]
Hi! I follow [the instructions](https://rnfirebase.io/#generating-ios-credentials) to add the file into my project, but when I see the `project.pbxproj` file in the line of the path says:

`0B4AAAF225925D0200C3BCEB /* GoogleService-Info.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; name = "GoogleService-Info.plist"; path = "../../../Downloads/GoogleService-Info.plist"; sourceTree = "<group>"; };`

Why says that path to me Downloads folder?

**Comment by mikehardy** [comment, marked as answer]
> "Copy items if needed" checkbox is enabled.

I think that step may have been missed. In essence *you* set that path. That is/was the path to the downloaded plist file. You really want it to be in your project though, thus "copy items if needed", or you can copy it yourself to the location you want then add it to your project

**Reply by MatiSera** [reply]
Maybe I forget to check that. But I solved the issue by deleting the path attribute. Thanks!

**Reply by Thaer-Sarakbi** [reply]
thank you so much @mikehardy
